packages/dcos-checks/extra/dcos-poststart-checks.timer

```diff
@@ -0,0 +1,4 @@
+[Unit]
+Description=DC/OS Checks Timer: timer for DC/OS Checks service
+[Timer]
+OnCalendar=minutely
```

Comment:
The default `AccuracySec` is 1 minute, meaning that this timer could fire anywhere from 0-120 seconds after the last run, which isn't what we want. We should probably set `AccuracySec` to something like 5 or 10 seconds, so that we get close to minute resolution while still giving systemd some flexibility on when to fire the timer for efficiency's sake.